### DISCUSSION TITLE
Question about v0.69 release notes

### DISCUSSION THREAD

**Discussion by brentonv** [moneytoo/Player#213] [General]
In your release notes for v0.69 it mentions 

> This build keeps using ExoPlayer 2.15.1 as there are still some unresolved issues in 2.16.0 as well as in the development (2.16.1) branches.

Could you please explain what the issues are?

Thanks

**Comment by moneytoo** [comment]
See the linked issues in [v0.67](https://github.com/moneytoo/Player/releases/tag/v0.67) and [v0.68](https://github.com/moneytoo/Player/releases/tag/v0.68).

**Reply by brentonv** [reply]
Thank you @moneytoo that's easy for me to follow🙂
